bin/completion-handlers.ts

```diff
@@ -20,50 +22,49 @@ export function setupCompletionForPackageManager(
 }
 
 export function setupPnpmCompletions(completion: Completion) {
-  completion.addCommand('add', 'Install a package', [], async () => []);
-  completion.addCommand('remove', 'Remove a package', [], async () => []);
+  completion.addCommand('add', 'Install a package', [], noopCompletion);
```

Comment:
i thought descriptions were being retrieved from the package managers themselves as we talked. we should not do that manually.